server/src/main/java/org/elasticsearch/action/datastreams/DataStreamsActionUtil.java

```diff
@@ -5,7 +5,8 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-package org.elasticsearch.datastreams.action;
+
+package org.elasticsearch.action.datastreams;
```

Comment:
Is this class move on purpose?

Reply:
Yes, I moved it so we can use it from both modules the `dlm` and the `data-streams`. If you would prefer to do something else, I am open to rework this.

Reply:
Could the `data-streams` module export this package to `dlm`? (DLM already depends on packages from `data-streams`)
We can keep this outside of `server` that way?